2024Robot_Team3484/src/main/cpp/subsystems/ShooterSubsystem.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "subsystems/ShooterSubsystem.h"
+
+#include <numbers>
+
+using namespace ShooterConstants;
+
+ShooterSubsystem::ShooterSubsystem() {
+
+    SH_Left_motor.RestoreFactoryDefaults();
+    SH_Right_motor.RestoreFactoryDefaults();
+
+    SH_Left_motor.SetInverted(MOTER_LEFT_INVERTED);
+    SH_Right_motor.SetInverted(!MOTER_LEFT_INVERTED);
+
+    SH_m_Left_pidController.SetP(P);
+    SH_m_Left_pidController.SetI(I);
+    SH_m_Left_pidController.SetD(D);
+    SH_m_Left_pidController.SetIZone(0);
+    SH_m_Left_pidController.SetFF(FF);
+    SH_m_Left_pidController.SetOutputRange(0, 1);
+
+    SH_m_Right_pidController.SetP(P);
+    SH_m_Right_pidController.SetI(I);
+    SH_m_Right_pidController.SetD(D);
+    SH_m_Right_pidController.SetIZone(0);
+    SH_m_Right_pidController.SetFF(FF);
+    SH_m_Right_pidController.SetOutputRange(0, 1);
+}
+
+void ShooterSubsystem::setSHRPM(units::revolutions_per_minute_t speed){
```

Comment:
nit: capitalize `set`.
not a nit: I suggest renaming the function to be a bit more descriptive of what is being set here. The name simply reads "Set RPM" which doesn't really tell anyone what this function is for.